packages/babel-parser/test/fixtures/flow/anonymous-function-types/bad_01/options.json

```diff
@@ -1,3 +1,8 @@
 {
+  "sourceType": "module",
+  "plugins": [
+    "jsx",
+    "flow"
+  ],
   "throws": "Unexpected token, expected \";\" (1:29)"
```

Comment:
Could you revert all these unrelated changes to the tests? :sweat_smile:

Reply:
Sure thing.